examples/msgpack/CMakeLists.txt

```
@@ -11,4 +11,4 @@ set (CMAKE_CXX_STANDARD 11)
 find_package(msgpack CONFIG REQUIRED)
 
 add_executable(container container.cpp)
-target_link_libraries(container msgpack::msgpack)
+target_link_libraries(container msgpackc-cxx)
```

although it makes no difference in this case, let's be explicit with the visibility of the linking

```suggestion
target_link_libraries(container PRIVATE msgpackc-cxx)
```